.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
```

Comment:
- [ ] Please use Microverse README template to create the readme file linter is just a tool it is not the main thing that your repo is about